server/network/aoprotocol.py

```diff
@@ -549,31 +591,36 @@ def net_cmd_ms(self, args):
                 self.client.send_ooc('You don\'t any areas!')
                 return
             text = ' '.join(part[1:])
+            send_args[4] = text
         elif text.startswith('/testify '): # Start a new testimony in this area.
             part = text.split(' ')
             text = ' '.join(part[1:]) # remove command
+            send_args[4] = text
             if not self.client.area.start_testimony(self.client, text):
                 return
             text = '~~-- ' + text + ' --'
+            send_args[4] = text
             color = 3 # orange
```

Comment:
don't you need to write the color to the recording as well

Reply:
no, because the recording doesn't ever replay the testimony title the way it was said, instead the person who issues the /examine command in IC says the title which is passed in plain text, and the orange color is automatically applied to that